src/wms/src/main/java/org/geoserver/wms/WMSRequests.java

```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Our copyright handling procedures do not demand anymore update of years in the header, just that a header needs to be there. No need for change, just letting you know for the future.